webapp/channels/src/components/user_settings/display/user_settings_theme/custom_theme_chooser/custom_theme_chooser.tsx

```diff
@@ -448,91 +457,112 @@ export class CustomThemeChooser extends React.PureComponent<Props, State> {
         );
 
         return (
-            <div className='appearance-section pt-2'>
+            <div
+                id='customThemesSection'
+                className='appearance-section pt-2'
+                aria-labelledby='customThemes'
+            >
                 <div className='theme-elements row'>
-                    <div
-                        ref={this.sidebarStylesHeaderRef}
-                        id='sidebarStyles'
-                        className='theme-elements__header'
-                        onClick={this.toggleSidebarStyles}
-                    >
-                        <FormattedMessage
-                            id='user.settings.custom_theme.sidebarTitle'
-                            defaultMessage='Sidebar Styles'
-                        />
-                        <div className='header__icon'>
-                            <i
-                                className='fa fa-plus'
-                                title={intl.formatMessage({id: 'generic_icons.expand', defaultMessage: 'Expand Icon'})}
-                            />
-                            <i
-                                className='fa fa-minus'
-                                title={intl.formatMessage({id: 'generic_icons.collapse', defaultMessage: 'Collapse Icon'})}
+                    <h4 className='theme-elements__header'>
+                        <button
+                            ref={this.sidebarStylesHeaderRef}
+                            id='sidebarStylesAccordion'
+                            onClick={this.toggleSidebarStyles}
+                            aria-expanded={false}
+                            aria-controls='sidebarStylesSection'
+                            aria-label='Sidebar Styles'
```

Comment:
These should be translatable by using `intl.formatMessage`. You can use the translation ID and default message from the `FormattedMessage` found below for that

Reply:
Actually, you don't need the `aria-label` on these buttons because the they'll get an accessible name from the text inside them automatically. You'll have to add `aria-hidden` to the expand and collapse icons inside of each of these as well, but once you've done that and removed the `aria-label`, the name of each component will be the translated text from the `FormattedMessage` inside of it